typesetting@v0.3.1 android cross compilation error

Hello!
I have a small project that uses [fyne](https://github.com/fyne-io/fyne) to create the user interface. I tried updating my project's dependencies and running the`fyne-cross android -image fyneio/fyne-cross-images:v1.3.1-android` build command on my **linux mint 22.2** machine (`Linux 6.8.0-90-generic x86_64`), but I got the following error:
```bash
go build -ldflags -w -trimpath -buildmode=c-shared -o /tmp/fyne-work-3929759310/lib/armeabi-v7a/libKeenetic_Mode_Switch.so github.com/streamdp/modeswitch failed: exit status 1
# github.com/go-text/typesetting/fontscan
/go/pkg/mod/github.com/go-text/typesetting@v0.3.1/fontscan/fontmap_cache_android.go:7:13: undefined: fmt
```
I checked this file and it indeed does not have any imports, but it probably should, since it uses **fmt.Errorf()** to create an error.

And of course, the build completed successfully with the previous version **typesetting@v0.3.0**.

--- a/fontscan/fontmap_cache_android.go
+++ b/fontscan/fontmap_cache_android.go
@@ -1,5 +1,7 @@
 package fontscan
 
+import "fmt"
+
 func platformCacheDir() (string, error) {
 	// There is no stable way to infer the proper place to store the cache
 	// with access to the Java runtime for the application. Rather than

